app/routes.js

```diff
@@ -144,8 +144,8 @@ var users = require('./models/users');
 
      var filepaths = require('./controller/filepaths');
      app.get('/api/v2/filepaths', filepaths.getAll);
-     app.get('/api/v2/filepaths/:id', filepaths.findById);
      app.get('/api/v2/filepaths/unassigned', filepaths.getAllUnassigned);
```

Comment:
Nice catch, I think that instead of moving it down which further obscures the issue, we should change that route name so that it's obvious that "unassigned" is not a route but an error. that way it should fall into Express handler for 404.

Reply:
If that's the case, I'd say we should look into query parameter, following the format of:
`/api/v2/filepaths?unassigned=true`

We're still requesting the resource `filepaths` but with a query filter. That saves you from creating a different REST resource or using POST when we're getting data from the db.

Reply:
That works for me.

Reply:
Done in 6429e5a